cmd/conformance-tester/pkg/scenarios/hetzner.go

```diff
@@ -84,20 +55,29 @@ func (s *hetznerScenario) APICluster(secrets types.Secrets) *apimodels.CreateClu
 
 func (s *hetznerScenario) Cluster(secrets types.Secrets) *kubermaticv1.ClusterSpec {
 	return &kubermaticv1.ClusterSpec{
+		ContainerRuntime: s.containerRuntime,
+		Features: map[string]bool{
+			kubermaticv1.ClusterFeatureExternalCloudProvider: true,
+		},
```

Comment:
Why do this for Hetzner only? It doesn't look like it's done for any other provider, and afaik there's code somewhere else setting the feature flags if CCM is supported.

Reply:
You are right and I even already considered that, as client_kube.go already does

```go
	if cloudcontroller.ExternalCloudControllerFeatureSupported(scenario.Datacenter(), cluster, version.NewFromConfiguration(c.opts.KubermaticConfiguration).GetIncompatibilities()...) {
		cluster.Spec.Features = map[string]bool{kubermaticv1.ClusterFeatureExternalCloudProvider: true}
		if cloudcontroller.ExternalCloudControllerClusterName(cluster) {
			cluster.Spec.Features[kubermaticv1.ClusterFeatureCCMClusterName] = true
		}
		if cluster.Spec.Cloud.VSphere != nil {
			cluster.Spec.Features[kubermaticv1.ClusterFeatureVsphereCSIClusterID] = true
		}
	}
```